yt/data_objects/static_output.py

```diff
@@ -403,10 +403,15 @@ def _guess_candidates(cls, base, directories, files):
         This function doesn't need to catch all possibilities, nor does it need
         to filter possibilities.
         """
+        # arguments named should be prefixed by a '_' if and only if they are not used
+        # in the function's body
+        # note that the original signature included a `directories` args
+        # that was not used in any subclass
```

Comment:
Should this comment disappear?

Reply:
yeah that should have been a GitHub comment. Thanks for pointing it out.